mako.5.scd

```diff
@@ -31,9 +31,9 @@ Empty lines and lines that begin with # are ignored.
 
 # BINDING OPTIONS
 
-Specify the action to take per button or touch event. Supported values are
-_none_, _dismiss_, _dismiss-all_, _dismiss-group_, _invoke-default-action_.
-
+Bindings allow to execute an action when an event is triggered. Supported
```

Comment:
Maybe "allow performing an action" instead of "execute" to avoid conflating with the exec action?

Reply:
Good idea